Makefile

```diff
@@ -37,16 +47,42 @@ TESSERACT_VERSION := 4.1.0
 TESSDATA_REPO = _best
 
 # Ground truth directory. Default: $(GROUND_TRUTH_DIR)
-GROUND_TRUTH_DIR := data/ground-truth
+GROUND_TRUTH_DIR := $(OUTPUT_DIR)-ground-truth
 
 # Max iterations. Default: $(MAX_ITERATIONS)
 MAX_ITERATIONS := 10000
 
+# Debug Interval. Default:  $(DEBUG_INTERVAL)
+DEBUG_INTERVAL := 0
+
 # Network specification. Default: $(NET_SPEC)
 NET_SPEC := [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c\#\#\#]
 
-# Normalization Mode - see src/training/language_specific.sh for details. Default: $(NORM_MODE)
-NORM_MODE = 2
+# Index for Layer to be replaced. Default: $(LAYER_APPEND_INDEX)
+LAYER_APPEND_INDEX := 5
+
+# Replace Layer Network specification. Default: $(LAYER_NET_SPEC)
+LAYER_NET_SPEC := [Lfx192 O1c1]
+
+# Language Type - Indic, RTL or blank. Default: '$(LANG_TYPE)'
+LANG_TYPE ?=
+
+# Normalization mode - 2, 1 - for unicharset_extractor and Pass through Recoder for combine_lang_model
+ifeq ($(LANG_TYPE),Indic)
+	NORM_MODE =2
+	RECODER =--pass_through_recoder
+	GENERATE_BOX_SCRIPT =generate_wordstr_box.py
+else
+ifeq ($(LANG_TYPE),RTL)
+	NORM_MODE =3
+	RECODER =--pass_through_recoder --lang_is_rtl
+	GENERATE_BOX_SCRIPT =generate_wordstr_box.py
+else
+	NORM_MODE =2
```

Comment:
@Shreeshrii I am no way sure that `NORM_MODE=2` is actually the best choice for Latin-type languages! I just wondered whether you have more insights.
@stweil Could you pls. share your experiences with this parameter?